Add region endpoint FAQ.md (#5152)
segmentio/segment-docs: src/connections/destinations/catalog/actions-segment-profiles/index.md

The Segment Profiles destination is not subject to API call or MTU costs. Any users or accounts created and updated by the Segment Profiles destination will not count towards your API call or MTU usage.

### Error loading options from Segment Profiles
Make sure you have a Public API token in your settings. If the Segment Profiles Destination doesn’t have a Public API token in the settings, the **Select Mappings** section fails to find any Profile (Engage) Spaces, which means you can't set your mappings. 

### Succesful syncs but no changes on profiles

> Make sure that the Endpoint Region setting matches the region of your workspace. If the region is properly set, and changes are still not applied, [contact Segment](https://segment.com/help/contact/){:target="_blank"}.

Reviewer: ```suggestion
Make sure that the Endpoint Region setting matches the region of your workspace. If the region is correct and you don't see any profile changes, [contact Segment](https://segment.com/help/contact/){:target="_blank"}.
```
